### PR TITLE
[AC-7228] Remove unused user roles

### PR DESCRIPTION
**Pull request by jonkiparsky**
Verify that there are no existing Program Role Grants with the user role "Senior Judge" or "Active Judge". 

This PR should merge after its sibling [accelerate PR](https://github.com/masschallenge/accelerate/pull/2420) or simultaneously with it.

### CODE REVIEW

**Comment by pbx** [comment]
Taking code review. Made a whitespace-cleanup commit.

File: accelerator/models/__init__.py [hunk 1]
```diff
@@ -6,11 +6,12 @@
 from .allocator import Allocator
 from .startup_role import StartupRole
 from .user_role import UserRole
+from accelerator_abstract.models.base_user_role import (
```
Comment (pbx) [inline accelerator/models/__init__.py]: Might as well collapse this down to a single line.

File: accelerator/tests/test_base_user_role.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+from __future__ import unicode_literals
```
Comment (pbx) [inline accelerator/tests/test_base_user_role.py]: No longer needed now that we're on Python 3.

File: accelerator/tests/test_base_user_role.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+from django.test import TestCase
+
+from accelerator.tests.factories import MemberFactory
```
Comment (pbx) [inline accelerator/tests/test_base_user_role.py]: Unused.

**Comment by jonkiparsky** [comment]
@pbx  Thanks, made those changes

**Pull request merged**